src/components/VLink.vue

```diff
@@ -21,19 +23,26 @@ export default defineComponent({
   props: {
     href: {
       type: String,
-      required: true,
-      validator: (v) => !['', '#'].includes(v),
```

Comment:
nit: I think we can keep the validator, otherwise accidental long-lived invalid `href` values won't ever get caught.